src/encoded/static/components/genome_browser.js

```diff
@@ -627,6 +627,7 @@ class GenomeBrowser extends React.Component {
             trackObj.displayLabels = false;
             trackObj.heightPx = labelLength > 0 ? (defaultHeight + (extraLineHeight * labelLength)) : defaultHeight;
             trackObj.expandedHeightPx = 140;
+            trackObj.fileFormatType = file.file_format_type;
```

Comment:
@emro Before, bigbed files was all understood as following the column format of UCSC. With this update, there are 6 recognizable Bigbed files-    BigBed (as defined by UCSC) , Broad Peak, Narrow Peak, Methyl, Tss Peak, Idr Peak. If all bigbed variants not part of this list default to bigbed as defined by UCSC